tensorflow/core/framework/tensor.h

```diff
@@ -35,6 +35,12 @@ limitations under the License.
 #include "tensorflow/core/platform/mem.h"
 #include "tensorflow/core/platform/types.h"
 
+#if !defined(PLUGGABLE_DEVICE_SUPPORTED) &&                                \
```

Comment:
Do you mean
```suggestion
#if !defined(PLUGGABLE_DEVICE_SUPPORTED_MACOS) &&                                \
```
?

Reply:
Yes, as discussed, I have created a separate Macro for this.